src/storage/storage_info.cpp

```diff
@@ -2,6 +2,6 @@
 
 namespace duckdb {
 
-const uint64_t VERSION_NUMBER = 33;
+const uint64_t VERSION_NUMBER = 34;
```

Comment:
As discussed I think a storage version increase could be avoided here - perhaps have another look?